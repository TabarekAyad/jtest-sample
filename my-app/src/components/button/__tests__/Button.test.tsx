import * as React from "react";
import * as renderer from "react-test-renderer";
import Button from '../button'
it("renders apply now correctly", () => {
  const tree = renderer.create(<Button name="hello"/>).toJSON();
  expect(tree).toMatchSnapshot();
});
