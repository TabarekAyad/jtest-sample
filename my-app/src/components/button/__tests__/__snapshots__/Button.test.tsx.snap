// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders apply now correctly 1`] = `
<button>
  hello
</button>
`;
